xcode/Subconscious/Shared/Components/AppView.swift

```diff
@@ -87,7 +87,6 @@ struct AppView: View {
         .onChange(of: self.scenePhase) { phase in
             store.send(.scenePhaseChange(phase))
         }
-        .sentryTrace("AppView")
```

Comment:
Too expensive?

Reply:
Too expensive _and_ not actually useful without forking out more money to sentry.